### ISSUE TITLE
Sequences with the same name

### ISSUE THREAD

**Issue by datcomputerguy** [whitesoup12/remote-falcon-tracker#121]
I have some sequences that I repeat in my sequence list and it appears that when I go use the Visible/Hidden button it only updates one of them.

**Comment by datcomputerguy** [comment]
Also noticed that the system didn't know how to handle deleting the duplicate sequences as well.  Not sure if this is on purpose or not.

**Comment by whitesoup12** [comment]
Not sure why you'd have sequences of the same name in RF. Sounds like it would just lead to confusion for you and/or your viewers.

**Comment by datcomputerguy** [comment]
I had to make some "in-between" sequences to appease my neighbors.  Basically after every main sequence there is this in-between sequence that plays that is calm and tame, and then after 2 minutes or whatever it plays the next main sequence unless the playlist is advanced.
So like I have the in-between sequences all set to hidden in remote falcon so that they don't show in the vote window.  I did a work around of just copy and pasting the "in-between" file sequence and adding a number after each one which works lol.
